### PR TITLE
MongoDB Driver

### PR DESCRIPTION
**Pull request by kolesnik**
mongodb structure fix

### CODE REVIEW

File: src/mongodb-driver.js [hunk 1]
```diff
@@ -23,30 +23,37 @@ export class MongoDbDriver {
...
+            startDate = startOfPeriod(date, period).format(),
+            collection = period.toString();
+
+
+        this.db.collection(period).findOne({[dateKey]: {'$exists': true}}, (err, doc) => {
+            if (err) {
+                console.log(err);
+            }
+
+            console.log(doc);
```
Comment (igor0ser) [inline src/mongodb-driver.js]: please remove console

File: src/mongodb-driver.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class MongoDbDriver {
 
     constructor(options) {
         this.options = options;
```
Comment (igor0ser) [inline src/mongodb-driver.js:12]: do we need this.options? we are using it only in constructor

Comment (kucherenko) [reply src/mongodb-driver.js:12]: this is common interface for al drivers

File: src/mongodb-driver.js [hunk 3]
```diff
@@ -23,30 +23,37 @@ export class MongoDbDriver {
...
+
         });
     }
 
     getCollection(type, date, period) {
         let dateKey = this._getDateKey(type, date, period),
-            startDate = startOfPeriod(date, period).toString();
-        return this.db.collection(this.collection).find({dateKey, startDate}).sort({startDate: 1});
+            startDate = startOfPeriod(date, period).toString(),
+            collection = period.toString();
```
Comment (kucherenko) [inline src/mongodb-driver.js:50]: I hope all data should be in one collection, not recommended use different collections for different periods

File: src/mongodb-driver.js [hunk 4]
```diff
@@ -31,12 +31,14 @@ export class MongoDbDriver {
...
-                this.db.collection(this.collection).insertOne({period: period, [dateKey]: data});
+                this.db.collection(this.collection).insertOne({
+                    period: period, startDate: startDate, [dateKey]: data
+                });
 
             } else {
 
                 this.db.collection(this.collection).findOneAndUpdate({period: period, [dateKey]: {'$exists': true}},
-                    {$set: {period: period, [dateKey]: data}});
+                    {$set: {period: period, startDate: startDate, [dateKey]: data}});
```
Comment (kucherenko) [inline src/mongodb-driver.js:41]: btw you can use following code
```
{period, startDate, dateKey, data}
```

**Pull request merged**